getting-started/orientation/index.md

```diff
@@ -20,15 +20,107 @@ The main parts of the OpenSpace user interface.
 
 
 :::{tip}
-**Removing Interface Elements**
+**Hiding Interface Elements**
+
+You can toggle the text in the Graphics Window on and off using the {kbd}`Shift` + {kbd}`F1` keys.
```

Comment:
```suggestion
You can toggle the text in the Graphics Window using the {kbd}`Shift+F1` keys.
```

1. "toggling" and "on and off' seems redundant
2. Can combine the keybind commands since it is a single one